docs/index.rst

```diff
@@ -53,6 +53,7 @@ Contents:
    usage
    backends/index
    api
+   apiimplementations
```

Comment:
I think `backends` would be a better name (also, I have a hard time reading words squashed together like this - my brain likes to change the letters when I see words I don't recognize).

Reply:
In fact, it looks like there are already `backends/index` with individual pages for each backend.

Reply:
Yeah, but the current `backends/index` (and the stuff below it) is really a completely different beast. A lot of it is implementation or usage notes.

Should I try to merge the two sets of pages? Or keep them separate and rename (Backend APIs and Backend Notes)? Or is there a middle ground?

Reply:
I think it would be OK to merge them.